validation/genesis/genesis-allocs_test.go

```diff
@@ -174,6 +175,11 @@ func testGenesisAllocs(t *testing.T, chain *ChainConfig) {
 	g, err := core.LoadOPStackGenesis(chainId)
 	require.NoError(t, err)
 
+	if chainId == uint64(1301) {
+		delete(g.Alloc, common.HexToAddress("0x1f98431c8ad98523631ae4a59f267346ea31f984"))
+		delete(g.Alloc, common.HexToAddress("0x5c69bee701ef814a2b6a3edd4b1652cb9cc5aa6f"))
+	}
```

Comment:
Still checking with governance about how to workaround the new predeploys. Now sure this is how we want to handle it

Reply:
These are preinstalls, not predeploys. The predeploys live in a particular address space `0x42...xx`

I think this is fine to merge given that its passing and we can ticket some follow up action items once we figure them out